addons/storyshots/package.json

```diff
@@ -20,8 +20,10 @@
     "babel-runtime": "^6.26.0",
     "glob": "^7.1.2",
     "global": "^4.3.2",
+    "jest-preset-angular": "^5.0.0",
     "jest-specific-snapshot": "^0.3.0",
     "prop-types": "^15.6.0",
+    "react-test-renderer": "^16.2.0",
```

Comment:
Can you please check that it works in a react 15 app?

Reply:
Would this block the merge? I presume this is the same for master?

Reply:
On master, it's a peer dep

Reply:
Maybe I can just limit it to the >= 15?

Reply:
What do you mean? It's not a peer dep. `>=15` will just install latest 16

Reply:
Yeah, right. I'll check it with 15 so. 🤦‍♂️